### PR TITLE
Closes Loading bar occassionally disappears during load after deleting data #1837

### PR DESCRIPTION
**Pull request by nosark**
Setting the default session.loading to true alleviates a race case that hits the else statement inside of session.getLoading().observe(this, new NonNullObserver<Boolean>() {} correcting the disappearing progressView and failing background transition that was occurring concurrently with the progressView issue. If this causes other issues I can find another look into other possible fixes :)

### CODE REVIEW

**Comment by mcomella** [comment]
Thanks for the PR and the research! I'll have time to take a look today or tomorrow.

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 1]
```diff
@@ -235,8 +235,7 @@ public void onChanged(@Nullable String url) {
             @Override
             public void onValueChanged(@NonNull Boolean loading) {
                 if (loading) {
-                    backgroundTransitionGroup.resetTransition();
-
+                     backgroundTransitionGroup.resetTransition();
```
Comment (mcomella) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: Please don't make changes to unrelated code – it makes it harder to track the changes in code over time with `git blame`!

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 2]
```diff
@@ -344,7 +343,6 @@ private void initialiseCustomTabUi(final @NonNull View view) {
         eraseContainer.removeView(sessions);
 
         final int textColor;
-
```
Comment (mcomella) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: Same.

**Comment by pocmo** [comment]
I'm also a bit concerned. While this might fix the issue here I'm not sure what the side effects are. In the past such patches have caused other bugs so we should take some time to investigate and test thoroughly.
